sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -347,6 +347,10 @@ querySpecification
        windows?)
     ;
 
+intoClause
+    : INTO tableIdentifier
```

Comment:
For example, the json data registed as table tbl_b is:
![json data](https://raw.githubusercontent.com/wuxianxingkong/storage/master/use_data.png)
@hvanhovell 
The Logical Plan of sql "SELECT a INTO tbl_a FROM tbl_b" is:
![json plan](https://raw.githubusercontent.com/wuxianxingkong/storage/master/print_json_plan.png)
The results match the expectations